
07/12/2020 08:02:01 - Falha interna da função <b>WriteDBUPDATEMACHINEDATA</b>
07/12/2020 08:02:01 - System.Data.SqlClient.SqlException (0x80131904): There is already an object named 'exe_MCONTROL_OEE_000033_Days' in the database.
   em System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   em System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   em System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   em System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   em System.Data.SqlClient.SqlCommand.RunExecuteNonQueryTds(String methodName, Boolean async, Int32 timeout, Boolean asyncWrite)
   em System.Data.SqlClient.SqlCommand.InternalExecuteNonQuery(TaskCompletionSource`1 completion, String methodName, Boolean sendToPipe, Int32 timeout, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   em System.Data.SqlClient.SqlCommand.ExecuteNonQuery()
   em MCONTROL_OEE.mod_OEE.WriteDBUPDATEMACHINEDATA(DataGridViewX dgvMAQUINA, String& strErro) na D:\VOESTALPINE - DEV\MCONTROL_OEE\MCONTROL_OEE\mod_OEE.vb:linha 744
ClientConnectionId:b61a1762-533d-4174-b635-556bb992359d
Error Number:2714,State:6,Class:16